infra/src/main/java/com/redhat/service/smartevents/infra/models/bridges/BridgeDefinition.java

```diff
@@ -7,14 +7,28 @@
 
 public class BridgeDefinition {
 
+    public static final String ENDPOINT_ERROR_HANDLER_TYPE = "endpoint";
+
     @JsonProperty("error_handler")
     private Action errorHandler;
 
+    @JsonProperty("resolved_error_handler")
+    private Action resolvedErrorHandler;
+
+    public static boolean isEndpointErrorHandlerAction(Action action) {
```

Comment:
@kostola Again, I'd suggest we move this onto the `ProcessingErrorService` as a method there. Perhaps something like `ProcessingErrorService.isEndpointErrorHandlerAction(Action action)`. 

It just helps to keep everything entirely encapsulated on that service. If we no longer want to support it, we could then simply delete the `package` and the `BridgeDefinition` doesn't need to be refactored.

Reply:
done :+1: